lib/path.js

```diff
@@ -1171,6 +1171,12 @@ const posix = {
    * @returns {string}
    */
   resolve(...args) {
+    if (args.length === 0) {
+      const cwd = posixCwd();
+      if (StringPrototypeCharCodeAt(cwd, 0) === CHAR_FORWARD_SLASH) {
```

Comment:
is this really faster than `if (cwd[0] === '/') {`?

Reply:
It used to be but I just checked with V8 13.6 and using the more intuitive notation is actually faster by now!

Reply:
It seems like it's not always like that. I do see a lot of improvements but also a lot of regressions, so I'll keep these changes out for now.

Reply:
Fair enough. Seems worth a larger separate exploration then.